### PR TITLE
Fix when collateral token is not in network config

### LINKED ISSUE #143: [FE] - Position List - Error when position's collateral token is not included in network config tokens

### PR DESCRIPTION
**Pull request by davidalbela**
Closes #143

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr146--conditionaltokensexplorer.review.gnosisdev.com

File: src/pages/PositionsList/index.tsx [hunk 1]
```diff
@@ -48,11 +48,12 @@ export const PositionsList = () => {
...
       cell: (row: Position) => {
-        return networkConfig ? (
-          <TokenIcon symbol={networkConfig.getTokenFromAddress(row.collateralToken).symbol} />
-        ) : (
-          row.collateralToken
-        )
+        try {
+          const token = networkConfig && networkConfig.getTokenFromAddress(row.collateralToken)
+          return <TokenIcon symbol={token.symbol} />
+        } catch {
```
Comment (mariano-aguero) [inline src/pages/PositionsList/index.tsx:54]: Good work, What about to add the `(error)`  parameter and display it in the console with `logger.error(error)` ? What do you think ?

Comment (lmcorbalan) [reply src/pages/PositionsList/index.tsx:54]: 👍

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr146--conditionaltokensexplorer.review.gnosisdev.com

**Pull request merged**